app/src/main/java/io/neurolab/fragments/MemoryGraphFragment.java

```diff
@@ -295,6 +296,8 @@ public void onActivityResult(int requestCode, int resultCode, @Nullable Intent d
     private void importData(String path) {
         if (permission) {
             if (StatisticsFragment.parsedData == null || parsedData == null) {
+                menu.getItem(1).setVisible(false);
```

Comment:
Can we use getitembyid instead? Because if we add any additional items to menu, this needs to be changed. Also we can create a function as it's same code in three places..

Reply:
Yeah, good suggestion. Actually as we have less items and we don't plan to add any more items to this toolbar (will get messy), i used this approach. And yup will make a function. Thanks.